integrations/spark/spark-validate-cleanse/spark-validate-cleanse-app/src/main/java/com/thinkbiganalytics/spark/datavalidator/CleansedRowResult.java

```diff
@@ -0,0 +1,36 @@
+package com.thinkbiganalytics.spark.datavalidator;
+
+/*-
+ * #%L
+ * thinkbig-spark-validate-cleanse-app
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.apache.spark.sql.Row;
+import java.io.Serializable;
+
+/**
+ * Created by cp186050 on 14/2/17.
+ */
+// Wrapper class to contain our cleansed row and its column validation results
```

Comment:
Please remove "Created by cp186050 on 14/2/17" and make this comment a JavaDoc. Also, documenting the class properties are encouraged but not required in this case.